### PR TITLE
feat: add slate

### PR DESCRIPTION
**Pull request by ferhatelmas**

### CODE REVIEW

File: slate/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# slate
+
+> [slate](https://github.com/lord/slate) in a docker image
+
+## develop
+
+> docker run --rm -v $PWD/docs:/slate/source -p 4567:4567 -it slate dev
+
+the preview will be available at http://127.0.0.1:5467
```
Comment (ekrem95) [inline slate/README.md]: `4567` or `5467` ?

File: slate/entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+if [ -z "$1" ]; then
+	echo "Deleting old 'build' folder"
+	rm -rf source/build
+
+	echo "Building static html doc to 'build' folder"
+	bundle exec middleman build --clean
+
+	echo "Copying result into 'build' folder"
```
Comment (ekrem95) [inline slate/entrypoint.sh]: isn't `cp -r build source/` copies `build` into `source` folder ?

Comment (ferhatelmas) [reply slate/entrypoint.sh]: yes, under source

File: slate/README.md (hunk 1 above)
Comment (ferhatelmas) [reply slate/README.md]: corrected

**Pull request merged**